stable/influxdb/values.yaml

```diff
@@ -56,6 +56,29 @@ persistence:
   accessMode: ReadWriteOnce
   size: 8Gi
 
+## Deploy InfluxDB Enterprise - License required
+##
+enterprise:
+  enabled: false
+  licensekey: {}
```

Comment:
Would be nice to add an option to provide the license key via a pre existing k8s secret

Reply:
That's a good idea. I'd like to leave as an improvement for a later PR, if that's okay with you.